app/models/concerns/audit_associations.rb

```diff
@@ -59,6 +45,12 @@ def audit_associations(*associations)
       end
     end
 
+    def normalize_associations(associations)
+      Array(associations).map do |association|
+        "#{association.to_s.singularize}_ids"
```

Comment:
if I understand this correctly, change to append `_ids` is required so that we later can tell whether the audited change is for association or not? If so, I think we could also check audited associations, which would make this change much smaller, but I think I'm fine with the change.

Reply:
we want the `audited_changes` to contain the `_ids` field so we can revert them if needed.

Reply:
Ok, makes sense, also that works with dirty cache